client/src/App/index.js

```diff
@@ -1,23 +1,24 @@
-import React from 'react';
-import logo from '../logo.svg';
+import React, { useState, useMemo } from 'react';
 import './App.css';
+import { Switch, Route } from 'react-router-dom';
+import LoginPage from '../pages/logIn';
+import IsAuthContext from '../components/context/isAuthContext';
 
 function App() {
+  const [isAuth, setIsAuth] = useState(false);
+
+  const value = useMemo(() => ({ isAuth, setIsAuth }), [isAuth, setIsAuth]);
+
   return (
     <div className="App">
       <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
+        <IsAuthContext.Provider value={value}>
+          <Switch>
+            <Route exact path="/login">
+              <LoginPage />
+            </Route>
+          </Switch>
+        </IsAuthContext.Provider>
```

Comment:
If You make the context like [this](https://github.com/GSG-G9/dentoro/pull/62/files#r609413886)
Then the app file will be smaller, you don't need here to import useState or useContext or isAuthContext, Just useAuth only :)
```js
import React from 'react';
import './App.css';
import { Switch, Route } from 'react-router-dom';
import LoginPage from '../pages/logIn';
import { useAuth } from '../components/context/isAuthContext';

function App() {
  const [isAuth] = useAuth();
  return (
    <div className="App">
      <header className="App-header">
        <Switch>
          <Route exact path="/login">
            <LoginPage />
          </Route>
          <Route exact path="/">
            {isAuth ? <h2>Logged</h2> : <h2>Not Logged</h2>}
          </Route>
        </Switch>
      </header>
    </div>
  );
}

export default App;

```

I make this line as an example only 
```js
          <Route exact path="/">
            {isAuth ? <h2>Logged</h2> : <h2>Not Logged</h2>}
          </Route>
```